jekyll/_cci2/orb-author-faq.md

```diff
@@ -94,7 +106,7 @@ steps:
     name: Install Homebrew (for Linux)
 ```
 
-## Command vs job
+## Command vs Job
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 100. You can fix by inserting `{: #command-vs-job }` to the next line

Reply:
Added